docs/source/conf.py

```diff
@@ -62,9 +62,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = u'1.0'
+version = u'2.0'
```

Comment:
@austinbyers can this pull from https://github.com/airbnb/streamalert/blob/f803613f839ac2b97284481ca451fcb6b27c73ea/stream_alert/__init__.py (also can you bump this version?)

Reply:
I'll go ahead and update that version, but I wasn't able to automatically pull it into the docs:

```python
# docs/source/conf.py
from stream_alert import __version__ as streamalert_version
# ...
version = streamalert_version
```

broke doc building:
```
$ sphinx-build -W docs/source docs/build
Running Sphinx v1.7.5

Configuration error:
There is a programable error in your configuration file:

Traceback (most recent call last):
  File ".../venv/lib/python2.7/site-packages/sphinx/config.py", line 161, in __init__
    execfile_(filename, config)
  File ".../venv/lib/python2.7/site-packages/sphinx/util/pycompat.py", line 150, in execfile_
    exec_(code, _globals)
  File ".../venv/lib/python2.7/site-packages/six.py", line 709, in exec_
    exec("""exec _code_ in _globs_, _locs_""")
  File "<string>", line 1, in <module>
  File "conf.py", line 23, in <module>
    from stream_alert import __version__ as streamalert_version
ImportError: No module named stream_alert
```

I also tried adding `__init__.py` everywhere to make the `docs` tree a Python package, same error

Reply:
sorry I should have added more context on how to achieve this - you shouldn't actually import the module, use you should use a method similar to what is recommended by `setuptools`: 

https://github.com/ThreatResponse/margaritashotgun/blob/master/setup.py#L12-L16

for reference, see # 1 here: https://packaging.python.org/guides/single-sourcing-package-version/